### DISCUSSION TITLE
首款大螢幕！小米手環7 Pro登台 售價1995元

### DISCUSSION THREAD

**Discussion by ABCDEFGHIJKLMNO12345678** [1101-GS4719-C/Python-Community-GS4719-C#760] [General]
隨者智慧手錶的日益進步 越來越多人會考率購入一隻 然而價錢也是一個很重要的考量 蘋果相較其它品牌不那麼親民 相信這次小米推出的產品能開開大家的眼界

https://tw.news.yahoo.com/%E9%A6%96%E6%AC%BE%E5%A4%A7%E8%9E%A2%E5%B9%95-%E5%B0%8F%E7%B1%B3%E6%89%8B%E7%92%B07-pro%E7%99%BB%E5%8F%B0-%E5%94%AE%E5%83%B91995%E5%85%83-025542562.html

**Comment by Yunhans** [comment]
希望智慧手錶的續航力可以再更久一點，不用每天都要充電

**Comment by danwan1027** [comment]
小米的價格相對親民，很適合當新手的入門款

**Comment by Pietra1226** [comment]
目前沒有使用過小米的產品，但看到這篇貼文讓我很想去買來試試看

**Comment by williamyen0623** [comment]
小米的東西真的平價又好用～

**Comment by manipulateed** [comment]
本來就有一隻小米手環，十分好用和價格便宜。

**Comment by evanchou1204** [comment]
比起apple watch，小米顯然便宜超級多。

**Comment by Doreen331** [comment]
我認為對我而言小米手環和apple watch的定位不太一樣，小米手環感覺比較像是工具類的手錶，可能運動的時候用來測一些身體相關的數據會用到。apple watch比較像是3c類工具，可能因為我是蘋果生態系的一員，如果我有apple watch能夠和手機連結之類的就會很方便，而且apple watch的價格確實需要被歸類到科技產品...

**Comment by Kotori0501** [comment]
謝謝分享
價格親民，因為體育課要維持一定的運動時間並做心率測試，所以現在非常需要買一隻來測我的跑步心率。

**Comment by ru0372572** [comment]
小米真的不錯~室友有一個小米檯燈有超多可以調的東西，亮度顏色高度都可以依照自己的需求調整，超讚。

**Comment by ching020721** [comment]
價格跟功能有很大的關係，需求不同購買的商品就不一樣，我本身對手錶沒有太大需求，但偶爾會覺得電子手錶很方便，apple watch的價格相對較高，若僅是需要運動與定位等功能，無須搭配手機連接使用，購買小米這支是很實惠的選擇~

**Comment by Zoe06060606** [comment]
一直以來都是用蘋果的產品 但這樣的價錢和性能真的很吸引人
